kafka/modules/kafka/2.3.0/module.yaml

```diff
@@ -8,7 +8,7 @@ envs:
     value: "amqstreams13-kafka-23-openshift-container"
 
 artifacts:
-  - md5: b5b3555df5969d06a9510b52d9e415c7
+  - md5: b5f5e2505ba2c287c7a077a4b4bf6807
```

Comment:
this MD5 seems to come from PNC (not Brew) and it's about the Kafka 2.3 zip for RHEL, not for OCP with all the dependencies.